### ISSUE TITLE
NLL Optimized?

### ISSUE THREAD

**Issue by GloryyrolG** [benjiebob/3D-Multibodies#3]
Hi Benjie,

It seems the implemented losses include an NLL which is not mentioned in the original paper if I understood correctly.

https://github.com/benjiebob/3D-Multibodies/blob/a9c493369d6f1942bff8787d6eb41914df5f89a1/src/model.py#L311

Thx & Regards,

**Comment by GloryyrolG** [comment]
As well as the depth loss,

https://github.com/benjiebob/3D-Multibodies/blob/a9c493369d6f1942bff8787d6eb41914df5f89a1/src/model.py#L313

**Comment by benjiebob** [comment]
Hi @GloryyrolG - thanks a lot for noticing this. I can confirm that *these losses were not used in the model released for the paper*. I added these losses for some last-minute and ultimately unused hacking prior to the NeurIPS deadline and forgot to remove before I uploaded the code. You can safely remove these losses.

Just FYI - I'd be a little careful using the training code at this point. I did a significant refactor removing some proprietary libraries in order to release the evaluation code. However, I haven't yet extensively tested the training scripts in this new repository so it's possible some other niggles like this exist. 

When I get some time, I'll push an update for this. 

Thanks again,
Ben
